hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HeapMemStoreLAB.java

```diff
@@ -206,6 +214,32 @@ private Chunk getOrMakeChunk() {
     }
   }
 
+  /** HBASE-14921
+   * Given a chunk ID return reference to the relevant chunk
+   * @return a chunk
+   */
+  public Chunk translateIdToChunk(int id) {
+    return chunkPool.translateIdToChunk(id);
+  }
+
+  /** HBASE-14921
+   * Give the ID of the Chunk from where last allocation took the bytes
+   * @return a chunk
+   */
+  public int getCurrentChunkId() {
+    return curChunk.get().getId();
+  }
+
+  /** HBASE-14921
```

Comment:
No need to spread the jira # everywhere, it's confusing. Whoever wants to track the history can do it via the repo.